lib/endpoint-notifier.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+// Simple echo server using websockets.
```

Comment:
No, that's what it is, the endpoint notifier just echos messages from one websocket to all other websockets. I'll add a better comment saying that.